e2e/cypress/support/okta_commands.js

```diff
@@ -0,0 +1,274 @@
+Cypress.Commands.add('oktaGetOrCreateUser', (user) => {
+    let userId;
+    cy.oktaGetUser(user.email).then((uId) => {
+        userId = uId;
+        if (userId == null) {
+            cy.oktaCreateUser(user).then((_uId) => {
+                userId = _uId;
+                cy.oktaAssignUserToApplication(userId, user);
+            });
+        } else {
+            cy.oktaAssignUserToApplication(userId, user);
+        }
+        return cy.wrap(userId);
+    });
+});
+
+Cypress.Commands.add('oktaAddUsers', (users) => {
+    let userId;
+    Object.values(users.regulars).forEach((_user) => {
+        cy.oktaGetUser(_user.email).then((uId) => {
+            userId = uId;
+            if (userId == null) {
+                cy.oktaCreateUser(_user).then((_uId) => {
+                    userId = _uId;
+                    cy.oktaAssignUserToApplication(userId, _user);
+                    cy.oktaDeleteSession(userId);
+                });
+            }
+        });
+    });
+
+    Object.values(users.guests).forEach((_user) => {
+        cy.oktaGetUser(_user.email).then((uId) => {
+            userId = uId;
+            if (userId == null) {
+                cy.oktaCreateUser(_user).then((_uId) => {
+                    userId = _uId;
+                    cy.oktaAssignUserToApplication(userId, _user);
+                    cy.oktaDeleteSession(userId);
+                });
+            }
+        });
+    });
+
+    Object.values(users.admins).forEach((_user) => {
+        cy.oktaGetUser(_user.email).then((uId) => {
+            userId = uId;
+            if (userId == null) {
+                cy.oktaCreateUser(_user).then((_uId) => {
+                    userId = _uId;
+                    cy.oktaAssignUserToApplication(userId, _user);
+                    cy.oktaDeleteSession(userId);
+                });
+            }
+        });
+    });
+});
+
+Cypress.Commands.add('oktaRemoveUsers', (users) => {
+    let userId;
+    Object.values(users.regulars).forEach((_user) => {
+        cy.oktaGetUser(_user.email).then((_uId) => {
+            userId = _uId;
+            if (userId != null) {
+                cy.oktaDeleteUser(userId);
+            }
+        });
+    });
+
+    Object.values(users.guests).forEach((_user) => {
+        cy.oktaGetUser(_user.email).then((_uId) => {
+            userId = _uId;
+            if (userId != null) {
+                cy.oktaDeleteUser(userId);
+            }
+        });
+    });
+
+    Object.values(users.admins).forEach((_user) => {
+        cy.oktaGetUser(_user.email).then((_uId) => {
+            userId = _uId;
+            if (userId != null) {
+                cy.oktaDeleteUser(userId);
+            }
+        });
+    });
+});
+
+Cypress.Commands.add('checkOktaLoginPage', () => {
+    cy.get('#okta-sign-in').should('be.visible');
+    cy.get('#okta-signin-username').should('be.visible');
+    cy.get('#okta-signin-password').should('be.visible');
+    cy.get('#okta-signin-submit').should('be.visible');
```

Comment:
Instead of checking element ID, please verify text by `cy.findByText('Submit').should('be.visible');`.  Same case to others if it makes sense.

Reply:
Used findByText in the cases where there is a text to denote the UX element with.